exp/api/remote/remote_api_test.go

```diff
@@ -295,3 +299,136 @@ func TestRemoteAPI_Write_WithHandler(t *testing.T) {
 		}
 	})
 }
+
+func TestSnappyDecodeMiddleware(t *testing.T) {
+	tLogger := slog.Default()
+
+	var actReq *writev2.Request
```

Comment:
This would perhaps allow us to remove comments "populated by ... "? (It's clear it's something we "get" from test.

```suggestion
	var gotRequest *writev2.Request
```

Reply:
fixed in 21da12f